README.md

```diff
@@ -44,7 +44,7 @@ Requires Node.js 6+ and npm 3+.
 #### Quick Start
 
 1. `git clone git@github.com:nteract/commuter.git`
```

Comment:
What do you think of doing this:

1. Enter in the terminal:
```
    git clone git@github.com:nteract/commuter.git
    npm i
    npm start
```

2. Open `http://localhost:3000`